sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOReadTest.java

```diff
@@ -377,124 +377,6 @@ public void processElement(ProcessContext c) throws Exception {
     p.run();
   }
 
-  @Test
-  public void testReadFromTableInteractive()
-      throws IOException, InterruptedException {
-    Table sometable = new Table();
-    sometable.setSchema(
-        new TableSchema()
-            .setFields(
-                ImmutableList.of(
-                    new TableFieldSchema().setName("name").setType("STRING"),
-                    new TableFieldSchema().setName("number").setType("INTEGER"))));
-    sometable.setTableReference(
-        new TableReference()
-            .setProjectId("non-executing-project")
-            .setDatasetId("somedataset")
-            .setTableId("sometable"));
-    sometable.setNumBytes(1024L * 1024L);
-    FakeDatasetService fakeDatasetService = new FakeDatasetService();
-    fakeDatasetService.createDataset("non-executing-project", "somedataset", "", "", null);
-    fakeDatasetService.createTable(sometable);
-
-    List<TableRow> records = Lists.newArrayList(
-        new TableRow().set("name", "a").set("number", 1L),
-        new TableRow().set("name", "b").set("number", 2L),
-        new TableRow().set("name", "c").set("number", 3L));
-    fakeDatasetService.insertAll(sometable.getTableReference(), records, null);
-
-    FakeBigQueryServices fakeBqServices = new FakeBigQueryServices()
-        .withJobService(new FakeJobService())
-        .withDatasetService(fakeDatasetService);
-
-    PTransform<PBegin, PCollection<TableRow>> readTransform;
-    BigQueryIO.TypedRead<TableRow> read =
-        BigQueryIO.readTableRows()
-            .from("non-executing-project:somedataset.sometable")
-            .withTestServices(fakeBqServices)
-            .withoutValidation()
-            .usingInteractivePriority();
-    readTransform = read;
-
-    PCollection<KV<String, Long>> output =
-        p.apply(readTransform)
-            .apply(
```

Comment:
Should we fix these tests (make sure that the request to BQ contain the correct priority) instead of deleting them ?

Reply:
These deleted tests are meaningless - they test reading from a table, whereas query priority only makes sense for reading from a query. I can add proper tests for fromQuery() but that'll take some time.

Reply:
I'm OK with getting these changes in without query tests for the property.